### PR TITLE
Update live-data-tutorial.md

### PR DESCRIPTION
**Pull request by Daniel-Nielsen**

### CODE REVIEW

File: collections/_ios/v3/live-data/live-data-tutorial.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ permalink: /ios/v3/live-data/live-data-tutorial
 
 In this tutorial you will learn to work with Live Updates / real-time data in MapsIndoors. It is recommended that you read the [Live Data Introduction](/ios/v3/live-data) before continuing.
 
-We will create a view controller displaying a map that shows some dynamic changes that are initiated from Live Data sources known by MapsIndoors. The test data are of the Occupancy Domain Type and the Position Domain Type.
+We will create a view controller displaying a map that shows some dynamic changes that are initiated from Live Data sources known by MapsIndoors. If you do not have a Live Data integration in place for your MapsIndoors project, you can use API key `2ae7d137162642618b5ce555` for demo and test purposes. The test data are of the Occupancy Domain Type and the Position Domain Type.
```
Comment (mbh-mapspeople) [inline collections/_ios/v3/live-data/live-data-tutorial.md:16]: The API Key - 2ae7d137162642618b5ce555 - points to a test solution... can we be sure that one exists and not not tampered with in the long run?

Comment (Daniel-Nielsen) [reply collections/_ios/v3/live-data/live-data-tutorial.md:16]: I am aware of that. But I think it's the only option we have right now... other than no demo data.

Comment (mbh-mapspeople) [reply collections/_ios/v3/live-data/live-data-tutorial.md:16]: Fair enough

**Review by mbh-mapspeople — APPROVED** [review]

File: collections/_ios/v3/live-data/live-data-tutorial.md (hunk 1 above)
Comment (Daniel-Nielsen) [reply collections/_ios/v3/live-data/live-data-tutorial.md:16]: I will create an internal issue to track this problem.

**Pull request merged**